buildenv/docker/mkdocker.sh

```diff
@@ -75,7 +75,7 @@ userid=1000
 version=unspecified
 
 # Frequently used commands.
-wget_O="wget --progress=dot:mega -O"
+wget_O="wget --no-check-certificate --progress=dot:mega -O"
```

Comment:
I suggest we should only use `--no-check-certificate` where it's needed, i.e. centos:6.

Reply:
I've added an overwrite of wget_O in the case of centos6 && x86_64 only.